lib/aws/active_job/sqs/configuration.rb

```diff
@@ -159,16 +158,21 @@ def initialize(options = {})
         # @api private
         attr_accessor :queues, :threads, :backpressure,
                       :shutdown_timeout, :client, :logger,
-                      :async_queue_error_handler,
-                      :retry_standard_errors
+                      :async_queue_error_handler
 
         # @api private
-        attr_writer :max_messages, :message_group_id, :visibility_timeout
+        attr_writer :max_messages, :message_group_id, :visibility_timeout,
+                    :poller_error_handler
 
         def excluded_deduplication_keys=(keys)
           @excluded_deduplication_keys = keys.map(&:to_s) | ['job_id']
         end
 
+        def poller_error_handler(&block)
+          @poller_error_handler = block if block
```

Comment:
Does rubocop not prefer block_given?

Reply:
apparently not (probably because I'm not using yield), but I'll change it.